src/c-writer.cc

```diff
@@ -796,6 +803,12 @@ void CWriter::Write(Type type) {
     case Type::I64: Write("u64"); break;
     case Type::F32: Write("f32"); break;
     case Type::F64: Write("f64"); break;
+    case Type::FuncRef:
+      Write("wasm_rt_funcref_t");
+      break;
+    case Type::ExternRef:
+      Write("wasm_rt_externref_t");
+      break;
```

Comment:
Ditto single line cases.

Reply:
The linter seems to force this format for newly added sections. :-/

Reply:
Can you wrap the whole case statement in `// clang-format off  ... // clang-format on` to avoid having two different styles.  We can re-visit the style question later perhaps.

Reply:
Done. I didn't know about that!